src/lsp.rs

```diff
@@ -64,6 +67,7 @@ impl<TLocalLinter: LocalLinter> Backend<TLocalLinter> {
             Some(&per_file_state.tree),
             "dummy_path",
             Default::default(),
+            SupportedLanguage::Rust,
```

Comment:
Still hard-coded Rust "at the periphery" in some places where I didn't feel like going ahead and wiring up the different-languages stuff eg here in the LSP server